src/Utils/request/useInfiniteQuery.ts

```diff
@@ -43,65 +39,27 @@ export function useInfiniteQuery<TData extends PaginatedResponse<TItem>, TItem>(
         : allItems;
```

Comment:
Hmm, shouldn't we call a Set here to get ensure unique items when dedplucateBy isn't specified 🤔

Reply:
<img width="385" alt="image" src="https://github.com/user-attachments/assets/1f391007-1ea2-4886-88cd-b6542c0a206a">

Reply:
we could either mandate the deDuplicateBy callback to be mandatory or not perform deduplication at all if undefined. I prefer the former.